### ISSUE TITLE
Qwen vl 7b instruct validation accuracy

### ISSUE THREAD

**Issue by achrafs758** [modelscope/ms-swift#2491]
I’m finetunning a qwen-vl-7b-instruct model on an ocr dataset.During the training ,i got around 95% as validation accuracy.I also save a checkpoint every 100 steps. When I load the checkpoint and do inference on the same validation set as during training, the accuracy is  much lower. Can someone explain in this context what the accuracy calculated refers to?

**Comment by z0o0ey** [comment]
please retry with latest swift and vllm, feel free to create a new issue if it the problem persists.
